src/components/PlantingSites/PlantingSitesList.tsx

```diff
@@ -36,15 +39,29 @@ export default function PlantingSitesList(props: PlantingSitesListProps): JSX.El
       fields: ['boundary', 'id', 'name', 'numPlantingZones', 'numPlots'],
       prefix: 'plantingSites',
       search: {
-        field: 'organization_id',
-        operation: 'field',
-        values: [organization.id],
+        operation: 'and',
+        children: [
+          {
+            field: 'organization_id',
+            operation: 'field',
+            values: [organization.id],
+          },
+          {
+            operation: 'field',
+            field: 'name',
+            type: 'Fuzzy',
+            values: [debouncedSearchTerm],
+          },
+        ],
       },
       count: 0,
     };
     const apiSearchResults = await search(params);
+    if (!debouncedSearchTerm) {
+      setPlantingSites(apiSearchResults);
```

Comment:
If I'm understanding correctly, is `plantingSites` the full list of planting sites when no search term is supplied?

Reply:
Yes, we need to save the full list to show or not the empty state, and in other place save the results when search term is applied, to show results in table